distributed/deploy/local.py

```diff
@@ -47,7 +47,9 @@ class LocalCluster(SpecCluster):
         Whether to use processes (True) or threads (False).  Defaults to True, unless
         worker_class=Worker, in which case it defaults to False.
     threads_per_worker: int
-        Number of threads per each worker
+        The number of threads in each worker's
+        :class:`python:ThreadPoolExecutor` where tasks are executed.
+        See also :doc:`dask:array-best-practices` for more information.
```

Comment:
This doesn't seem like the right place to point to array best practices docs.

```suggestion
```

I'm also wondering if this is still an issue since recent versions of `distributed` should limit `numpy`s concurrency by default

https://github.com/dask/distributed/blob/5dccad4c0b719a57b53404b02e5666cf23b119fd/distributed/distributed.yaml#L191-L194

In https://github.com/dask/dask/issues/9588 you mentioned you're using `dask=2022.05.0` -- I'm curious if you run into the same thread oversubscription issue with the latest `dask=2022.10.0` release

Reply:
> This doesn't seem like the right place to point to array best practices docs.

How about `Notes` instead? It would be nice to have it in this class docstring somewhere so that people thinking about threads can find complementary information. I would have found the link I think if it had been on the same page as `threads_per_worker` somewhere...

> I'm also wondering if this is still an issue since recent versions of distributed should limit numpys concurrency by default

Based on a similar suggestion/question earlier from @ncclementi I checked and indeed it's [fixed for me in 2022.10.0](https://github.com/dask/dask/issues/9588#issuecomment-1287088812) :)